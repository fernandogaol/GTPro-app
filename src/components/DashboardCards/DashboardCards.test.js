// import React from 'react';
// import renderer from 'react-test-renderer';
// import DashboardCards from './DashboardCards';

// it('renders correctly', () => {
//   const tree = renderer.create(<DashboardCards />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
