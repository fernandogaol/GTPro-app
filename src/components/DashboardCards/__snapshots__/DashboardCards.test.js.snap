// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="DashboardCards"
>
  <div>
    <textarea />
    <button>
      Add List +
    </button>
  </div>
</section>
`;
